packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -210,4 +210,13 @@ export class WorkflowRepository extends Repository<WorkflowEntity> {
 	async findByActiveState(activeState: boolean) {
 		return this.findBy({ active: activeState });
 	}
+
+	async findWithFields(candidateIds: string[], { fields }: { fields: string[] }) {
```

Comment:
There seem to already be `findByIds`. We could add an optional `select` param for it and it

Reply:
Good idea, done.